lib/taiko.js

```diff
@@ -3338,9 +3338,17 @@ const evaluate = async (selector, func) => {
 };
 
 const validate = () => {
-  if (!dom || !page) {
+  if (!_client) {
     throw new Error('Browser or page not initialized. Call `openBrowser()` before using this API');
   }
+  if (_client._ws.readyState > 1) {
+    if (chromeProcess && chromeProcess.killed) {
+      throw new Error(
+        'Browser is closed due to `closeBrowser()` or crash. Try reopening by `openBrowser()`',
+      );
+    }
+    throw new Error('Connection to browser lost');
```

Comment:
Suggestion:

`Connection to browser lost. This probably isn't a problem with Taiko, inspect logs for possible causes.`